exampleCourse/questions/fibonacciEditorWithR/question.html

```diff
@@ -0,0 +1,30 @@
+<pl-question-panel>
+  <p>
+    The Fibonacci numbers are 1, 1, 2, 3, 5, 8, ..., where each
+    number is the sum of the two before it.
+    \[
+    F_n = F_{n-1} + F_{n-2}
+    \]
+  </p>
+  <p>
+    Write an <b><i>R</i></b> function <tt>fib</tt> that takes a number <tt>n</tt>
+    and returns the n<sup>th</sup> Fibonacci number.
```

Comment:
As a further improvement over the current Python question, we should be more explicit about numbering (0-indexed versus 1-indexed) by specifying `... the n<sup>th</sup> Fibonacci number, starting from <tt>n = 1</tt>. For example, <tt>fib(3) = 2</tt>.`